ambry-store/src/main/java/com/github/ambry/store/StoreMessageReadSet.java

```diff
@@ -159,12 +161,12 @@ public void close() {
    */
   void doPrefetch(long relativeOffset, long size) throws IOException {
     long sizeToRead = Math.min(size, getMessageInfo().getSize() - relativeOffset);
-    prefetchedData = ByteBuffer.allocate((int) sizeToRead);
-    getChannel().read(prefetchedData, offset.getOffset() + relativeOffset);
+    prefetchedData = PooledByteBufAllocator.DEFAULT.ioBuffer((int) sizeToRead);
```

Comment:
Just want to double check:

We prefer direct buffer here. I looked at ioBuffer() function, it returns direct buffer if `isDirectBufferPooled`. How to know value of `isDirectBufferPooled`?

Reply:
netty has a jvm property to control that, but by default, in linux box, it's enabled.